hr_attendance_report_theoretical_time/wizards/recompute_theoretical_attendance_views.xml

```diff
@@ -4,7 +4,7 @@
     <record id="act_wizard_recompute_theoretical_attendance" model="ir.actions.act_window">
         <field name="name">Recompute Theoretical Attendances</field>
         <field name="res_model">recompute.theoretical.attendance</field>
-        <field name="view_type">form</field>
+
```

Comment:
don't leave an empty line, just remove the one you want to

Reply:
> don't leave an empty line, just remove the one you want to

done